doc/changes/v1.8.rst

```diff
@@ -129,7 +129,7 @@ New features
 - Add support for storing Fourier coefficients in :class:`mne.time_frequency.Spectrum`,
   :class:`mne.time_frequency.EpochsSpectrum`, :class:`mne.time_frequency.SpectrumArray`,
   and :class:`mne.time_frequency.EpochsSpectrumArray` objects, by `Thomas Binns`_. (`#12747 <https://github.com/mne-tools/mne-python/pulls/12747>`__)
-
+- :meth:`~mne.io.Raw` and :meth:`~mne.Epochs.save` now return the path to the saved file(s), by `Victor Ferat`_. (`#12811 <https://github.com/mne-tools/mne-python/pull/12811>`__)
```

Comment:
The changelog system has changed. The file you modified is automatically generated on release (1.8 is the current `stable` version). Instead, create a file `12811.newfeature.rst` in `doc/changes/devel`.